docs/eventing/sink/kafka-sink.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Apache Kafka Sink"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure Apache Kafka Sink.
+
+## Prerequisites
+
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
+
+## Installation
+
+1. Install the Kafka controller:
+
+    ```bash
+    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    ```
+
+1. Install the Kafka Sink data plane:
```

Comment:
```suggestion
1. Install the KafkaSink data plane:
```

I'd use either "KafkaSink data plane" or "Kafka sink data plane", whichever is correct. Sink should only be capitalized as part of a specific object, not as a general term.